pmd-core/src/main/java/net/sourceforge/pmd/renderers/CodeClimateRenderer.java

```diff
@@ -45,9 +46,9 @@ public CodeClimateRenderer() {
     }
 
     private static String getPmdDeveloperURL() {
-        String url = "http://pmd.github.io/pmd-" + PMD.VERSION + "/customizing/pmd-developer.html";
-        if (PMD.VERSION.contains("SNAPSHOT") || "unknown".equals(PMD.VERSION)) {
-            url = "http://pmd.sourceforge.net/snapshot/customizing/pmd-developer.html";
+        String url = "https://pmd.github.io/pmd-" + PMDVersion.VERSION + "/customizing/pmd-developer.html";
+        if (PMDVersion.isSnapshot() || PMDVersion.isUnknown()) {
+            url = "https://pmd.github.io/latest/customizing/pmd-developer.html";
```

Comment:
This URL is a 404. Would it be `https://pmd.github.io/latest/pmd_devdocs_development.html` ?

Reply:
Good catch! Thanks!